charts/incubator/alfresco-dbp-infrastructure/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: A Helm chart Providing the required Infrastructure deployments for the Alfresco DBP. This includes nginx-ingress, activemq, alfresco-api-gateway, alfresco-keycloak, and the content services volume.
+name: alfresco-dbp-infrastructure
```

Comment:
This should just be named `alfresco-infrastructure`.  We should not have any references to DBP in this GitHub repo.

Reply:
Ok.  There will be a couple of places that I need to update this. Will make the changes.